genesis/genesis_test.go

```diff
@@ -22,6 +22,12 @@ func TestTestnetGenesis(t *testing.T) {
 	b0, _, _, err := gene.Build(state.NewStater(db))
 	assert.Nil(t, err)
 
+	id := gene.ID()
+	name := gene.Name()
+
+	assert.Equal(t, id, thor.Bytes32{0x0, 0x0, 0x0, 0x0, 0xb, 0x2b, 0xce, 0x3c, 0x70, 0xbc, 0x64, 0x9a, 0x2, 0x74, 0x9e, 0x86, 0x87, 0x72, 0x1b, 0x9, 0xed, 0x2e, 0x15, 0x99, 0x7f, 0x46, 0x65, 0x36, 0xb2, 0xb, 0xb1, 0x27})
```

Comment:
`thor.MustParseBytes32("0x000000000b2bce3c70bc649a02749e8687721b09ed2e15997f466536b20bb127")` might be better for reading